python/lsst/cp/pipe/ptc.py

```diff
@@ -968,6 +968,7 @@ def errFunc(p, x, y):
                 msg = (f"\nSERIOUS: Not enough data points ({len(meanVecFinal)}) compared to the number of"
                        f"parameters of the PTC model({len(parsIniPtc)}). Setting {ampName} to BAD.")
                 self.log.warn(msg)
+                lenNonLinPars = self.config.polynomialFitDegreeNonLinearity - 1
```

Comment:
If anything, shouldn't this be `+1` not `-1`? If I fit an x^2 _i.e._ 2nd order poly, I'd expect 3 coeffs, not 1, right?